test_runner/performance/pageserver/pagebench/test_large_slru_basebackup.py

```diff
@@ -80,10 +80,6 @@ def setup_tenant_template(env: NeonEnv, n_txns: int):
         "gc_period": "0s",  # disable periodic gc
         "checkpoint_timeout": "10 years",
         "compaction_period": "0s",  # disable periodic compaction
-        "compaction_threshold": 10,
```

Comment:
This is just an unrelated code clean up, right? (just for my understanding)

Reply:
Yeah, these are the default settings.